lib/matplotlib/axis.py

```diff
@@ -1097,6 +1108,9 @@ def get_tightbbox(self, renderer):
         else:
             return None
 
+    def get_tick_padding(self):
+        return self.majorTicks[0].get_tick_padding()
```

Comment:
should this be `max(self.majorTicks[0].get_tick_padding(), self.minorTicks[0].get_tick_padding()))` ? 

Never underestimate pathological users....